client/my-sites/plans/jetpack-plans/jetpack-complete-page/items-included/utils.ts

```diff
@@ -0,0 +1,27 @@
+import {
+	PRODUCT_JETPACK_BACKUP_T1_MONTHLY,
+	JETPACK_BACKUP_PRODUCT_LANDING_PAGE_URL,
+	JETPACK_ANTI_SPAM_PRODUCT_LANDING_PAGE_URL,
+	JETPACK_SCAN_PRODUCT_LANDING_PAGE_URL,
+	JETPACK_SEARCH_PRODUCT_LANDING_PAGE_URL,
+	PRODUCT_JETPACK_ANTI_SPAM,
+	PRODUCT_JETPACK_BOOST,
+	PRODUCT_JETPACK_CRM,
+	PRODUCT_JETPACK_SCAN,
+	PRODUCT_JETPACK_SEARCH,
+	PRODUCT_JETPACK_SOCIAL_ADVANCED,
+	PRODUCT_JETPACK_VIDEOPRESS,
+} from '@automattic/calypso-products';
+
+const PRODUCT_URL_MAP: { [ key: string ]: string } = {
+	[ PRODUCT_JETPACK_BACKUP_T1_MONTHLY ]: JETPACK_BACKUP_PRODUCT_LANDING_PAGE_URL,
+	[ PRODUCT_JETPACK_BOOST ]: 'https://jetpack.com/boost/',
+	[ PRODUCT_JETPACK_SCAN ]: JETPACK_SCAN_PRODUCT_LANDING_PAGE_URL,
+	[ PRODUCT_JETPACK_SEARCH ]: JETPACK_SEARCH_PRODUCT_LANDING_PAGE_URL,
+	[ PRODUCT_JETPACK_ANTI_SPAM ]: JETPACK_ANTI_SPAM_PRODUCT_LANDING_PAGE_URL,
+	[ PRODUCT_JETPACK_SOCIAL_ADVANCED ]: 'https://jetpack.com/social/',
+	[ PRODUCT_JETPACK_VIDEOPRESS ]: 'https://jetpack.com/features/writing/video-hosting/',
```

Comment:
Although this link works: `https://jetpack.com/features/writing/video-hosting/`... (because it ends up redirecting to https://jetpack.com/videopress/), Should we change this link to point directly to https://jetpack.com/videopress/ ?  That way we avoid any redirecting?

Also, should we add some new landing page constants to the `@automattic/calypso-products` package?  Like for Jetpack Boost, Jetpack Social, and Jetpack VideoPress?  Or maybe even in a follow-up PR?, but if in a follow-up PR, we should probably create the task somewhere so it can be followed and not get lost.

Reply:
Thanks! Good idea to include these. Updated.